www/app/HardwareController.js

```diff
@@ -415,6 +440,12 @@ define(['app'], function (app) {
 					Mode5 = relaycount;
 				}
 				var password = encodeURIComponent($("#hardwarecontent #divlogin #password").val());
+				if (text.indexOf("eHouse") >= 0) 
+					{
+					if (password == "") {
+								ShowNotify($.t('Please enter ASCI password - 6 characters'), 2500, true);
+								}
```

Comment:
Please change this to "Please enter a Password!"
We do not want a new translation line when not needed
It is also ASCII

Reply:
hi, 
I removed all files that should'nt be there but there are still on github repository.  
HOW CAN I DO IT OTHER WAY THAN DELETE WHOLE REPOSITORY AND FORK IT Again???

I've removed all unused header files in repository & modify acording to your suggestions.


Regards, 

Robert Jarzabek